packages/app/package.json

```diff
@@ -46,13 +46,14 @@
     "@backstage/plugin-techdocs-react": "^1.0.5",
     "@backstage/plugin-user-settings": "^0.5.0",
     "@backstage/theme": "^0.2.16",
+    "@internal/backstage-plugin-rhacm": "*",
+    "@internal/plugin-sre-portal": "^0.1.0",
```

Comment:
Let's not version this for now and please rename it to `@internal/backstage-plugin-web-terminal`

Reply:
Resolved